src/ui/public/notify/__tests__/notifier.js

```diff
@@ -185,6 +197,60 @@ describe('Notifier', function () {
     });
   });
 
+  describe('#custom', function () {
+    let customNotif;
+
+    beforeEach(() => {
+      customNotif = notifier.custom(customText, customParams);
+    });
+
+    afterEach(() => {
+      customNotif.clear();
+    });
+
+    it('has a custom function to make notifications', function () {
+      expect(notifier.custom).to.be.defined;
```

Comment:
`expect(notifier.custom).to.be.a('function');` would more accurately assert what the test description asserts.